addon-test-support/helpers.js

```diff
@@ -45,6 +53,10 @@ export function closeFlatpickrDate(selector) {
  * @function clearFlatpickrDate
  */
 export function clearFlatpickrDate(selector) {
+  if (selector instanceof HTMLElement) {
+    selector._flatpickr.clear(); // eslint-disable-line
```

Comment:
Can we make these set `flatpickrInput` and then we can keep the rest of the logic as is and not return here?